ci/jenkins/Jenkinsfile_unix_gpu

```diff
@@ -39,7 +39,7 @@ core_logic: {
     custom_steps.compile_unix_full_gpu(),
     custom_steps.compile_unix_cmake_mkldnn_gpu(),
     custom_steps.compile_unix_cmake_gpu(),
-    custom_steps.compile_unix_tensorrt_gpu(),
+    // custom_steps.compile_unix_tensorrt_gpu(),
```

Comment:
Probably the ticket link should be mentioned in a comment

Reply:
I've updated the description to this ticket - I hope that this PR won't even be merged, if there fix works I guess we can ignore it. If it gets merged I can rebase the fix and re-enable - it will be like it never happened o_O